projects/sbb-esta/angular-business/src/lib/contextmenu/contextmenu/contextmenu.component.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  AfterContentInit,
+  Component,
+  ContentChild,
+  OnInit,
+  ViewChild,
+  ViewEncapsulation
+} from '@angular/core';
+import { DropdownComponent, DropdownTriggerDirective } from '@sbb-esta/angular-public';
+
+@Component({
```

Comment:
Configure changeDetection: ChangeDetection.OnPush for performance
Check if that breaks anything (it should not, since this is not a form element)